diff --git a/analisador-genealogico/app.py b/analisador-genealogico/app.py
index c432869..09249a2 100644
--- a/analisador-genealogico/app.py
+++ b/analisador-genealogico/app.py
@@ -11,9 +11,7 @@ app = Flask(__name__)
 app.secret_key = 'f@milyse@rch_dna_edition_v16'
 
 UPLOAD_FOLDER = "uploads"
-STATIC_FOLDER = "static"
 os.makedirs(UPLOAD_FOLDER, exist_ok=True)
-os.makedirs(STATIC_FOLDER, exist_ok=True)
 
 
 # --- Rota Principal ---
